pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -1162,6 +1162,70 @@ func TestValidatePipelineResults_Failure(t *testing.T) {
 	}
 }
 
+func TestFinallyTaskResultsToPipelineResults_Failure(t *testing.T) {
```

Comment:
please add an example where the expected error is nil

Reply:
just to clarify, does this mean to write a `success` test?

Reply:
yes